web/src/components/auth/resetPasswordConfirm.js

```diff
@@ -0,0 +1,13 @@
+import { Container } from "semantic-ui-react";
```

Comment:
I know it's weird. But, convention within the app would have us import and re-export "Container" through the "ui index"  and then import it from `ui`. This applies even if we haven't modified the component. The idea is to filter all of our components through the Custom UI folder, to 
1) track them in a central place and
2) make them as agnostic as possible

Same suggestion applies to the imports in `web/src/components/auth/resetPassword.js`

Reply:
Thanks for the reminder!